cmake/external/mklml.cmake

```diff
@@ -40,7 +40,7 @@ IF(WIN32)
     SET(MKLML_SHARED_LIB          ${MKLML_LIB_DIR}/mklml.dll)
     SET(MKLML_SHARED_IOMP_LIB     ${MKLML_LIB_DIR}/libiomp5md.dll)
 ELSE()  
-    SET(MKLML_VER "mklml_lnx_${TIME_VERSION}" CACHE STRING "" FORCE)
+    SET(MKLML_VER "VsErf_mklml_lnx_${TIME_VERSION}" CACHE STRING "" FORCE)
```

Comment:
Please add a comment to show this is a temporary mklml lib including erf, like 
`TODO(intel-huying)`?

Reply:
Done